test/README.md

```diff
@@ -0,0 +1,37 @@
+CSI Baremetal operator e2e testing
+---------------------
```

Comment:
Please move file to `docs` folder

Reply:
moved and renamed with `e2e_testing_on_kind`